.github/workflows/build-test-release.yml

```diff
@@ -47,6 +47,15 @@ jobs:
           fossa test --debug
         env:
           FOSSA_API_KEY: ${{ secrets.FOSSA_API_KEY }}
+      - name: Check for failed steps for jobs with continue-on-error
```

Comment:
Does this mean that we will
not be able to merge the PR if FOSSA fails?

Reply:
It does not, until we include it in `needs` of `all-checks`. I added it here just to avoid the confusion later when we want to restrict it more